pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1588,6 +1616,9 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
           await this.kubeBackend.deleteIncompatibleData(kubernetesVersion);
         }
 
+        await this.progressTracker.action('Configuring openresty', 50, this.configureOpenResty(config));
+        await this.startService('openresty');
```

Comment:
Rather than manually starting the service, it might be good to change the `containerd` / `docker` service file to add a dependency instead. (So that if the service accidentally dies it can get restarted due to needing `containerd`.)

Reply:
I had done this before, but ran into issue, but those may have been due to the double-backgrounding. I've added it back.

Done